### DISCUSSION TITLE
Has this use case Call-Center been removed from the Quickstarts?

### DISCUSSION THREAD

**Discussion by kentbill** [TimefoldAI/timefold-solver#736] [General]
I found that the call center example is missing in Quickstarts' use-cases. Has it been removed? I think it is a good example for demonstrating real-time planning.

https://github.com/TimefoldAI/timefold-quickstarts/tree/7d9205eba53dd430d565183db67c7f79d96e2492/use-cases

**Comment by triceo** [comment]
Hello @kentbill and thank you for your question.

Yes, it has been removed. The quickstarts are undergoing some changes, and the call center (together with vaccination scheduling) was one that no longer fits. That said, you can always find it in the Git history if you want to look it up, so no information was really lost.

**Comment by dvag-joerg-winter** [comment]
@triceo 
Hi, was the removal related to changes in surrounding- or quickstart-technology or to changes in timefold itself ?
Is there any chance to revive it ? It is the perfect fit for our use-case evaluation !

Best
